types/cucumber/index.d.ts

```diff
@@ -261,4 +268,32 @@ declare namespace cucumber {
     export class JsonFormatter extends Formatter {
     }
 
+    // Functions - Aliases since v2.3.0
```

Comment:
documentation says these are aliases of `defineStep` but I don't see that in this file.

Additionally, if they're all aliases, a better scheme would be

```ts
export function defineStep(pattern: RegExp | string, options: StepDefinitionOptions, code: StepDefinitionCode): void;
export function defineStep(pattern: RegExp | string, code: StepDefinitionCode): void;

export var Given: typeof defineStep;
export var When: typeof defineStep;
export var Then: typeof defineStep;
```

or even have a `DefineStep` interface - but any way you go about it, it's better to avoid duplication